### ISSUE TITLE
Add ability to search or filter keyword

### ISSUE THREAD

**Issue by shauncopley** [apfister/SDMXExplorer#2]
@apfister it would be great if you could add a function to search/filter for key words when setting dataflows and dimensions. Some of the lists have hundreds of selections and currently you need to manually scroll through the list to select a variable.

**Comment by apfister** [comment]
agreed. this one is on the roadmap just not entirely sure how to best accomplish it.

i was thinking to have a local "index" of some sorts that brings together the keywords from all data flows across providers. Then a separate "search" dialog/window could help you search through that index and find what you are looking for. Once found, the user could click on a "apply" or "use dataflow" to automatically populate the main pane with provider, agency, & dataflow ids.

open to thoughts.
